src/components/crossForm/CrossForm.vue

```diff
@@ -670,12 +654,19 @@ export default {
     },
     setClaimCost() {
       const { currentConfig: { isRsk } = {} } = this.sharedState
-      const web3 = isRsk ? this.sharedState.ethWeb3 : this.sharedState.rskWeb3
+      const web3 = isRsk ? this.sharedState.sideWeb3 : this.sharedState.rskWeb3
       web3.eth.getGasPrice().then(gasPrice => {
         const costInWei = new BigNumber(ESTIMATED_GAS_AVG).multipliedBy(gasPrice)
-        this.claimCost = costInWei.shiftedBy(-18).toString()
+        this.claimCost = `${costInWei.shiftedBy(-18).toString()} ${isRsk ? 'ETH' : 'RBTC'}`
```

Comment:
maybe move 18 to a constant called CLAIM_DECIMALS or something like that?

Reply:
Also use the currency from the Network config and not jarcoded as ETH and RBTC

Reply:
Done at https://github.com/rsksmart/tokenbridge-ui/commit/2945d716db8fe08c8b0e21aa2a9d3af81afe7978